<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S1_swath_merge</class>
 <widget class="QWidget" name="S1_swath_merge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>哨兵一号子带拼接</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/swath_merge.png</normaloff>:/QtWidgetsApplication3/bin/icon/swath_merge.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>干涉相位拼接</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>选择工程</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_project"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>IW1节点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_IW1_node"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>IW1数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_IW1_data"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1,6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>IW2节点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_IW2_node"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>IW2数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_IW2_data"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1,6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>IW3节点</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_IW3_node"/>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>IW3数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_IW3_data"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>目标节点名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dstNode"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="7,3">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>SAR图像拼接</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
